packages/framework/src/Model/Mail/MailTemplateConfiguration.php

```diff
@@ -74,11 +74,14 @@ public function addMailTemplateVariables(
         $this->mailTemplateVariables[$mailTemplateSlug] = $mailTemplateVariables;
     }
 
-    protected function registerOrderStatusMailTemplates(): void
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Mail\MailTemplateVariables
+     */
+    protected function addOrderStatusMailTemplateVariables(): MailTemplateVariables
     {
-        $orderStatusMailTemplate = new MailTemplateVariables('', self::TYPE_ORDER_STATUS);
+        $mailTemplateVariables = new MailTemplateVariables('', self::TYPE_ORDER_STATUS);
 
-        $orderStatusMailTemplate
+        return $mailTemplateVariables
```

Comment:
nice work :) 

the original issue mentions also the documentation. Is it possible to do it? 😇